### PR TITLE
[ Chore ] Move server to .ts

### PR DESCRIPTION
**Pull request by kewinzaq1**

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -44,16 +44,16 @@ app.use((_, res, next) => {
 
 app.use(
   helmet({
+    referrerPolicy: { policy: 'same-origin' },
```
Comment (kewinzaq1) [inline server.ts:47]: Before the placement of these rules was incorrect

**Comment by dev-xo** [comment]
Hello @kewinzaq1, I appreciate the efforts and the Pull Request.

Could you explain where you got the references for the implementation, or the steps of it?
- I will review the pull request, clean it a bit, and this will probably be a merge.

**Comment by kewinzaq1** [comment]
> Hello @kewinzaq1, I appreciate the efforts and the Pull Request.
> 
> Could you explain where you got the references for the implementation, or the steps of it?
> 
> * I will review the pull request, clean it a bit, and this will probably be a merge.
> 
> Solves: #232



> Hello @kewinzaq1, I appreciate the efforts and the Pull Request.
> 
> Could you explain where you got the references for the implementation, or the steps of it?
> 
> * I will review the pull request, clean it a bit, and this will probably be a merge.
> 
> Solves: #232

HI @dev-xo I did it based on The Epic Stack https://github.com/epicweb-dev/epic-stack

**Comment by dev-xo** [comment]
This is about to be merged @kewinzaq1.
- Do you know if this will impact deployment somehow?

**Review by dev-xo — APPROVED** [review]

**Comment by dev-xo** [comment]
Made a first deployment attempt. Although everything seems to work as expected in terms of deployment, the server is not able to initialize the app, and not sure why, as the run command has not changed.

Will give it another try later, in order to recheck this.
- Will also try to deploy the main repo without this PR's effects.

**Comment by kewinzaq1** [comment]
> Made a first deployment attempt. Although everything seems to work as expected in terms of deployment, the server is not able to initialize the app, and not sure why, as the run command has not changed.
> 
> Will give it another try later, in order to recheck this.
> 
> * Will also try to deploy the main repo without this PR's effects.
> 
> Moved `execa` to `dependencies`, instead of `devDependencies` (in a private repository), in the same way Epic Stack seems to have it configured.
> 
> * After deployment, this still doesn't seem to work either.
> 
> Leaving this PR open, as probably someone else will be curious about it, and we can keep discussing it, looking for a fix or for ways to handle this.

Thanks for the information. I'll check this

**Pull request closed without merging**